cmd/sebak/cmd/run.go

```diff
@@ -286,6 +326,9 @@ func runNode() {
 	var g run.Group
 	{
 		nr, err := sebak.NewNodeRunner(flagNetworkID, localNode, policy, nt, isaac, st)
+		conf := sebak.NewNodeRunnerConfiguration()
+		conf.SetINIT(timeoutINIT).SetSIGN(timeoutSIGN).SetACCEPT(timeoutACCEPT).SetALLCONFIRM(timeoutALLCONFIRM)
+		// nr.SetConf(conf)
```

Comment:
Dead code?

Reply:
This code will be used in next pr. I left it for notifying setup point. But I'm going to remove in this pr.